OpenRA.Mods.Common/Traits/BotModules/BaseBuilderBotModule.cs

```diff
@@ -128,6 +128,9 @@ public class BaseBuilderBotModuleInfo : ConditionalTraitInfo
 		[Desc("When should the AI start building specific buildings.")]
 		public readonly Dictionary<string, int> BuildingDelays = null;
 
+		[Desc("AI stop producing new building when its cash lower than this")]
+		public readonly int ProductionMinCashRequirement = 500;
```

Comment:
```suggestion
		public readonly int MiniumCash = 500;
```
?

Reply:
I don't know if just `MinimumCash` is better, because we may later have some function about the silos, which is also related to cash.